sdk/core/core/internal/az_config_internal.h

```diff
@@ -21,6 +21,7 @@ enum {
 enum {
   _az_TIME_SECONDS_PER_MINUTE = 60,
   _az_TIME_MILLISECONDS_PER_SECOND = 1000,
+  _az_TIME_MICROSECONDS_PER_MILLISECOND = 1000,
```

Comment:
Do we really need this? Especially if we're not going to have 64-bit integer resolution

Reply:
I know, but let's get rid of the 64 bits, and update this as part of that PR.